doc/design/compiler-environment-support.md

```diff
@@ -17,7 +17,6 @@ to:
  - Rhino
  - Nashorn
  - Internet Explorer
```

Comment:
I think that just IE is vast (http://babeljs.io/docs/usage/caveats/#internet-explorer). Do you know which versions we could support officialy? Edge?